### PR TITLE
Update dependencies and handle octokit upgrade

### LINKED ISSUE #329: Update not running after install, works from visual studio
For the past couple months, I've been having no issues with Squirrel.Windows, but recently (can't say for sure when it started, but I'm on 0.9.4 now), the application no longer updates after being installed.

When running from Visual Studio (in Release or Debug) it works fine.  The code in use is: https://github.com/amweiss/vigilant-cupcake/blob/master/VigilantCupcake/MainForm.cs#L507 which is nearly a copy/paste from the example posted here.  Any ideas what could cause the difference?  I'm trying to find a good way to attach a debugger and get symbols working post-install.

Fiddler shows the request going out to the releases URL, and no exceptions are bubbling up.

### PR DESCRIPTION
**Pull request by amweiss**
Hopefully fixes #329

### CODE REVIEW

File: src/SyncReleases/SyncImplementations.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ internal class SyncImplementations
 
             await releases.ForEachAsync(async release => {
                 // NB: Why do I have to double-fetch the release assets? It's already in GetAll
-                var assets = await client.Release.GetAssets(nwo.Item1, nwo.Item2, release.Id);
+                var assets = await client.Release.GetAllAssets(nwo.Item1, nwo.Item2, release.Id);
```
Comment (amweiss) [inline src/SyncReleases/SyncImplementations.cs]: This is due to https://github.com/octokit/octokit.net/releases/tag/v0.10.0

File: .nuget/packages.config [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="ILRepack" version="1.25.0" />
+  <package id="ILRepack" version="1.26.0" />
```
Comment (flagbug) [inline .nuget/packages.config]: Seems like there's gone something wrong here

Comment (amweiss) [inline .nuget/packages.config]: Hm, good call, let me go try to fix that

**Comment by amweiss** [comment]
As a note, after building the library with these changes, I dropped the .dlls into an installed app and it started working again, so hopefully this covers the current issues with strong naming.

**Comment by paulcbetts** [comment]
Thanks @amweiss

**Pull request merged**